lua/src/buildfile

```diff
@@ -0,0 +1,28 @@
+# Use an immediate import of `liblua%lib{lua}` to be able
+# to export it for backwards compatibility as `lua%lib{lua}`.
+# For installation, it must be stated
+# as a prerequisite to this directory.
+# Unfortunately, this also affects operations, such as `clean`,
+# to also affect `liblua` package when applied to `lua` package.
```

Comment:
nit: reflow comment to textwidth (with a maximum width of 80);


```suggestion
# Use an immediate import of `liblua%lib{lua}` to be able to export it for
# backwards compatibility as `lua%lib{lua}`. For installation, it must be
# stated as a prerequisite to this directory. Unfortunately, this also affects
# operations, such as `clean`, to also affect `liblua` package when applied to
# `lua` package.
```

nit: grammar *(suggestion here is best-effort; change as needed)*
```suggestion
# Import `liblua%lib{lua}` immediately to export it as `lua%lib{lua}` for
# backward compatibility. As for the installation case, declare it as a
# prerequisite for this directory. Note that this will affect operations like
# `clean`, which in turn will impact the `liblua` package when applied to the
# `lua` package.
```